docs/hazmat/primitives/mac/hmac.rst

```diff
@@ -28,10 +28,13 @@ of a message.
     .. doctest::
 
         >>> from cryptography.hazmat.primitives import hashes, hmac
+        >>> import os
+        >>> key = os.urandom(32)
         >>> h = hmac.HMAC(key, hashes.SHA256())
         >>> h.update(b"message to hash")
-        >>> h.finalize()
-        b'#F\xdaI\x8b"e\xc4\xf1\xbb\x9a\x8fc\xff\xf5\xdex.\xbc\xcd/+\x8a\x86\x1d\x84\'\xc3\xa6\x1d\xd8J'
+        >>> signature = h.finalize()
+        >>> # A signature is a 256-bit digest. E.g.
+        >>> # b'\xcb\x07\xe7.\x86&\xc3\t\xf8w\xb7\x8a1\xbe3\xa5JQH*5\x93\xf7H\xa0\x10\x80\xa8\x89\xf9vY'
```

Comment:
We want our doctests to actually test against values to confirm that they work and we haven't broken the API without realizing we need to update the docs. In this case you'll need to provide a fixed key for that, but you can add a comment explaining that users should generate and store their own key using `os.urandom` and load that in their own code.

Reply:
@reaperhulk I totally agree with you. And now a fixed key is added to code example. Also, I guess there is no need to add a comment about `os.urandom` since being addressed in previous instruction.